packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "50.2.2",
+  "version": "50.2.3",
```

Comment:
This should me `minor`. `v50.3.0`

Reply:
Ohhhhh okay I see the label on your PR now: https://github.com/department-of-veterans-affairs/component-library/pull/1604

So what made that `minor`? Because it's a variant?

Reply:
yeah I think I did minor because it causes a "Non-breaking or backwards compatible component API change".

There's also [a couple of other minor flags](https://github.com/department-of-veterans-affairs/component-library/pulls?q=is%3Apr+is%3Aclosed+is%3Amerged+sort%3Aupdated-desc):

![Screenshot 2025-05-28 at 1 12 54 PM](https://github.com/user-attachments/assets/54c9c2d0-3dbe-40d6-b302-e361cd9219a8)